src/xcode/ENA/ENA/Source/SceneDelegate.swift

```diff
@@ -49,8 +49,8 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate, RequiresAppDepend
 		let window = UIWindow(windowScene: windowScene)
 		self.window = window
 
-		#if UITESTING
-		if let isOnboarded = UserDefaults.standard.string(forKey: "isOnboarded") {
+		#if DEBUG
+		if isUITesting, let isOnboarded = UserDefaults.standard.string(forKey: "isOnboarded") {
```

Comment:
Is it on purpose that only the `isOnboarded` launch argument is exclusive to the UI tests now, and the others are also available in `DEBUG`?

Reply:
That's to distinguish unit from ui tests which has previously been via `UITESTING`. Without the `isUITesting` a unit test would use this `if` branch in case "isOnboarded" is set in the UserDefaults. This was not the case in the previous implementation.

Reply:
Thats true, my concern was more about the two following launch argument, that weren't available outside of UI tests before, but are now.

Reply:
Which ones? There are tons of **undocumented** arguments and environment variables buzzing around. This is also a problem that needs to be tackled but in a different story. Previously there where two schemes (ENATests & ENAUITests) that drifted apart. Now there is one common place to test things. If we screw up with launch parameters, we hopefully get inconsistencies within the tests.